fluent-plugin-logdna.gemspec

```diff
@@ -5,10 +5,10 @@ Gem::Specification.new do |s|
   s.name        = 'fluent-plugin-logdna'
   s.version     = '0.3.1'
   s.date        = Date.today.to_s
-  s.summary     = 'LogDNA plugin for Fluentd'
-  s.description = 'Fluentd plugin for supplying output to LogDNA.'
-  s.authors     = ['Edwin Lai']
-  s.email       = 'edwin@logdna.com'
+  s.summary     = 'LogDNA Plugin for FluentD'
+  s.description = 'FluentD Plugin for Supplying Output to LogDNA.'
+  s.authors  = ['Matt March, Mike Del Tito']
```

Comment:
should this just be "LogDNA, Inc" ? seems more standard/common and we're doing that elsewhere

https://github.com/logdna/logger-node/blob/239bb567270cef6944fe6cbfd61479033c9d9a2a/package.json#L31-L34

Reply:
Done!